packages/boot/tools/supports.ts

```diff
@@ -615,6 +619,13 @@ export const makeSwingsetTestKit = async (
       console.log('🧻');
       return i;
     },
+
+    async inboundVTransferEvent(params: BuildVTransferEventParams) {
+      await runUtils.queueAndRun(
+        () => inbound(BridgeId.VTRANSFER, buildVTransferEvent(params)),
+        true,
+      );
+    },
```

Comment:
this strikes me as too high level for `bridgeUtils`. The name also doesn't convey how it's different from simply `inbound(BridgeId.VTRANSFER, buildVTransferEvent(params)),`

WDYT of,
```suggestion
    async runInbound(...args: Parameters<typeof inbound>>) {
      await runUtils.queueAndRun(
        () => inbound(...args),
        true,
      );
    },
```

If we want better autocomplete we could improve the underlying `inbound` type to look at the first arg to narrow the rest.

Reply:
Sure, `Parameters<typeof inbound>` is just `...any[]`, so I narrowed to ask for `BridgeId` for the first argument.